components/org.wso2.carbon.identity.scim.common/src/main/java/org/wso2/carbon/identity/scim/common/internal/SCIMCommonComponent.java

```diff
@@ -81,6 +82,8 @@ protected void activate(ComponentContext ctx) {
             }
         } catch (CharonException e) {
             logger.error("Error in reading information from identity tables at SCIMCommonComponentStartup.", e);
+        } catch (UserStoreException e) {
```

Comment:
Why can't we catch a throwable and then log it.

Reply:
Fixed